index.js

```diff
@@ -42,7 +42,10 @@ function init() {
     const restoredNotificationsCheckboxValue = localStorage.getItem('notifications');
     formCheckboxElem.checked = JSON.parse(restoredNotificationsCheckboxValue);
     const restoredLang = localStorage.getItem('select-language');
-    changeLanguage(restoredLang);
+    const defaultLanguage = 'en';
```

Comment:
it is better to move this const outside this function, so it'll never be called more then one time